assets/js/blocks/customer-account/index.tsx

```diff
@@ -28,3 +29,13 @@ registerBlockType( metadata, {
 		return null;
 	},
 } );
+
+registerBlockVariation( 'woocommerce/customer-account', {
+	name: 'woocommerce/customer-account',
+	title: __( 'Customer account', 'woo-gutenberg-products-block' ),
+	isDefault: true,
+	attributes: {
+		...metadata.attributes,
+		iconClass: 'account-icon',
```

Comment:
I would prefix this class with `wc-block-`, following [the guidelines](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/contributors/contributing/coding-guidelines.md#prefixing). This makes it more difficult that in the future another plugin or theme might break our styles. Something like: `wc-block-customer-account__account-icon`.